### PR TITLE
Authors

### PR DESCRIPTION
**Pull request by scopatz**
Checklist
* [ ] Added a ``news`` entry

no news entry needed, no code changed

### CODE REVIEW

File: .authors.yml [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+  github: sodre
+- name: Christopher J. Wright
+  email: cjwright4242gh@gmail.com
+  alternate_emails:
+  - cjwright4242@gmail.com
+  aliases:
+  - christopher
+  num_commits: 50
+  first_commit: 2017-01-05 19:25:16
+  github: mariusvniekerk
```
Comment (CJ-Wright) [inline .authors.yml]: Is this correct?
```suggestion
  github: CJ-Wright
```

**Comment by scopatz** [comment]
Whoops sorry

File: .authors.yml [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+  email: widenetservices@gmail.com
+  num_commits: 9
+  first_commit: 2018-08-05 23:33:03
+- name: Isuru Fernando
+  email: isuru.11@cse.mrt.ac.lk
+  alternate_emails:
+  - isuruf@gmail.com
+  num_commits: 257
+  first_commit: 2017-01-08 22:03:03
+  github: mariusvniekerk
```
Comment (CJ-Wright) [inline .authors.yml]:
```suggestion
  github: isuruf
```

File: .authors.yml [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+- name: John Kirkham
+  email: jakirkham@gmail.com
+  alternate_emails:
+  - kirkhamj@janelia.hhmi.org
+  aliases:
+  - John Kirkham
+  - jakirkham
+  num_commits: 267
+  first_commit: 2016-03-17 21:05:53
+  github: minrk
```
Comment (CJ-Wright) [inline .authors.yml]:
```suggestion
  github: jakirkham
```

File: .authors.yml [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+  first_commit: 2015-07-24 07:12:10
+  github: rhattersley
+- name: Filipe Fernandes
+  email: ocefpaf@gmail.com
+  aliases:
+  - Filipe
+  - ocefpaf
+  num_commits: 26
+  first_commit: 2015-07-27 17:51:00
+  github: isuruf
```
Comment (CJ-Wright) [inline .authors.yml]:
```suggestion
  github: ocefpaf
```

File: .authors.yml [hunk 5]
```diff
@@ -0,0 +1,391 @@
...
+  alternate_emails:
+  - isuruf@gmail.com
+  num_commits: 257
+  first_commit: 2017-01-08 22:03:03
+  github: isuruf
+- name: Julian Rüth
+  email: julian.rueth@fsfe.org
+  num_commits: 9
+  first_commit: 2019-01-14 22:53:11
+  github: saraedum
```
Comment (CJ-Wright) [inline .authors.yml]: ?

File: .authors.yml [hunk 6]
```diff
@@ -0,0 +1,391 @@
...
+  num_commits: 1
+  first_commit: 2016-08-19 21:04:51
+  github: msarahan
+- name: Matthew Craig
+  email: mattwcraig@gmail.com
+  aliases:
+  - Matt Craig
+  num_commits: 6
+  first_commit: 2015-07-27 10:10:56
+  github: jakirkham
```
Comment (CJ-Wright) [inline .authors.yml]: ?

File: .authors.yml [hunk 7]
```diff
@@ -0,0 +1,391 @@
...
+- name: hajapy
+  email: hajapy@users.noreply.github.com
+  num_commits: 2
+  first_commit: 2019-08-19 20:33:42
+  github: hajapy
+- name: Peter Williams
+  email: peter@newton.cx
+  num_commits: 4
+  first_commit: 2017-04-23 09:00:59
+  github: jakirkham
```
Comment (CJ-Wright) [inline .authors.yml]: ?

File: .authors.yml [hunk 8]
```diff
@@ -0,0 +1,391 @@
...
+- name: Dougal J. Sutherland
+  email: dougal@gmail.com
+  github: dougalsutherland
+  num_commits: 36
+  first_commit: 2017-09-13 18:29:41
+- name: Anthony Scopatz
+  email: scopatz@gmail.com
+  num_commits: 88
+  first_commit: 2015-07-22 22:51:27
+  github: hmaarrfk
```
Comment (CJ-Wright) [inline .authors.yml]:
```suggestion
  github: scopatz
```

**Comment by CJ-Wright** [comment]
Thank you!

File: .authors.yml [hunk 9]
```diff
@@ -0,0 +1,391 @@
...
+  first_commit: 2019-07-31 17:05:02
+  github: thewtex
+- name: John Kirkham
+  email: jakirkham@gmail.com
+  alternate_emails:
+  - kirkhamj@janelia.hhmi.org
+  aliases:
+  - John Kirkham
+  - jakirkham
+  num_commits: 8
```
Comment (isuruf) [inline .authors.yml]: Is this correct?

Comment (scopatz) [reply .authors.yml]: hmmm, rever's commit counting seems to be off in some way.  Not sure what is going on here, but it is an issue with rever

Comment (scopatz) [reply .authors.yml]: Note that this isn't used for anything other than author ordering

Comment (jakirkham) [reply .authors.yml]: WDYT about ordering authors by surname?

**Pull request merged**